### PR TITLE
Sync between guide and cli command generation code

### PR DESCRIPTION
**Pull request by DPK101**
When running the command "ember g model rental" with ember-cli 3.10.1

### CODE REVIEW

**Comment by jenweber** [comment]
Hi! Thanks for this PR! We have an app that we keep in sync with the tutorial. Could you also PR the change to the demo app? This is the file to change https://github.com/ember-learn/super-rentals/blob/master/app/models/rental.js

Once that has a PR, we'll merge them at the same time. Thanks!

**Comment by DPK101** [comment]
Hi Jen!

Please find the another PR at [https://github.com/ember-learn/super-rentals/pull/126](https://github.com/ember-learn/super-rentals/pull/126)

Thank you

File: guides/release/tutorial/ember-data.md [hunk 1]
```diff
@@ -50,6 +51,7 @@ export default DS.Model.extend({
 });
 ```
 
+Note that we have used ES6 [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) with const declaration.
```
Comment (jenweber) [inline guides/release/tutorial/ember-data.md]:
```suggestion
If you are curious about that `{ Model }` syntax above, it is called ES6 [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) with `const` declaration, and it is a feature of JavaScript.
```

**Comment by jenweber** [comment]
Thank you! The Super Rentals PR is merged! However it looks like CI is failing on this PR with a spell check for "const." If you click "commit suggestion" here on this page, my edit will fix that problem by wrapping `const` in backticks. There is also a suggested wording change, to make it more welcoming to people who are just learning JavaScript.

File: guides/release/tutorial/ember-data.md [hunk 2]
```diff
@@ -50,6 +51,7 @@ export default DS.Model.extend({
 });
 ```
 
+If you are curious about that `{ Model }` syntax above, it is called ES6 [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) with `const` declaration, and it is a feature of JavaScript.
```
Comment (efx) [inline guides/release/tutorial/ember-data.md]: @DPK101 great find!
You can actually use destructuring with any kind of variable declaration:
```js
const pet = {
  details: {
    name: 'fred'
  }
};

let { details: { name } } = pet;
console.log(name);
```
Perhaps we can remove the  `with const declaration` qualification. As an aside, we could consider adding destructuring as an entry in the [glossary](https://github.com/ember-learn/guides-source/blob/master/guides/release/glossary/web-development.md).

Comment (DPK101) [reply guides/release/tutorial/ember-data.md]: Nice catch @efx 
I have done the correction.

File: guides/release/tutorial/ember-data.md [hunk 3]
```diff
@@ -50,6 +51,7 @@ export default DS.Model.extend({
 });
 ```
 
+If you are curious about that `{ Model }` syntax above, it is called ES6 [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) and it is a feature of JavaScript.
```
Comment (locks) [inline guides/release/tutorial/ember-data.md]:
```suggestion
If you are curious about the `const { Model } = DS;` syntax above, it is called [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) and it is a feature of JavaScript.
```

Comment (locks) [reply guides/release/tutorial/ember-data.md]: @DPK101 what do you think of these changes? :)

**Comment by jenweber** [comment]
This will be merged into the Guides via https://github.com/ember-learn/guides-source/pull/927/files which incorporates feedback from @locks 

Thanks all for your help!

**Pull request closed without merging**